serverless/aws/terraform/src/main/scala/sttp/tapir/serverless/aws/terraform/model.scala

```diff
@@ -0,0 +1,18 @@
+package sttp.tapir.serverless.aws.terraform
+import io.circe.Printer
+import io.circe.syntax._
+import sttp.model.Method
+import sttp.tapir.serverless.aws.terraform.AwsTerraformEncoders._
+
+case class AwsTerraformApiGateway(routes: Seq[AwsApiGatewayRoute]) {
+  def toJson()(implicit options: AwsTerraformOptions): String = {
+    val gateway = this
```

Comment:
why do we need this? :)

Reply:
right it's not actually required, I thought it was since IDE is showing an error in this line